src/mixins/object_interactivity.mixin.js

```diff
@@ -72,6 +72,9 @@
      * everything would resolve to a single point and a pythagorean theorem for the distance
      * @private
      */
+
```

Comment:
See note in controls file about these not working properly. This is where the touch-code for controls lives but I haven't investigated further. (Currently the touch-area for the play/pause control is a bit small - still usable but noticeable)